.github/workflows/google_cloud_http_target_task.js

```diff
@@ -0,0 +1,66 @@
+const { CloudTasksClient } = require('@google-cloud/tasks');
+const yenv = require('yenv')
+const { v4: uuidv4 } = require('uuid');
+
+const parsedDocument = yenv('.github/workflows/gql-perf-arch-github_workflow.yml')
```

Comment:
Is it possible to use process.env.COMMIT_HASH and likewise for all other env variables? we might not need yenv dependency.

Reply:
cc

Reply:
Yes, you were right. The change is done.